logging/google/cloud/logging/_environment_vars.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2017 Google Inc.
```

Comment:
@lukesneeringer We should probably move most env. vars. out of `google-cloud-core`. `gcloud.environment_vars` made sense for DRY and documentation purposes, but that's not really true anymore after the subpackage split.